### DISCUSSION TITLE
h1 tags in a package's readme.md can't be center aligned like other heading tags

### DISCUSSION THREAD

**Discussion by davidmurdoch** [npm/feedback#169] [Website] [answered]
If you publish a readme like:

```html
<h1 align="center">
  center aligned text
</h1>
<h2 align="center">
  center aligned text
</h2>
```

The h1 tag will be left aligned, while the h2 tag will be correctly center aligned.

The npm website has the rule `.markdown > h1 { text-align: left; }` in the CSS, which is already the default alignment... so this rule only prevents intentional center alignment. H1 tags can be center aligned this way in both GitHub and Docker; so it'd be nice if npm would align with their behavior.

**Comment by MylesBorins** [comment, marked as answer]
Hey @davidmurdoch thanks for opening this! I've created an internal bug to see if the engineering team can pick this up.

**Reply by lumaxis** [reply]
@davidmurdoch This should be fixed now! Would you mind checking the package page you were having issues with again and see if everything looks good now?

**Reply by davidmurdoch** [reply]
It works! Thanks for the quick turn around!
